src/sources/docker_logs/mod.rs

```diff
@@ -712,12 +808,23 @@ impl EventStreamBuilder {
     }
 }
 
-fn add_hostname(mut event: LogEvent, host_key: &str, hostname: &Option<String>) -> LogEvent {
+fn add_hostname(
+    mut log: LogEvent,
+    host_key: &str,
+    hostname: &Option<String>,
+    log_namespace: LogNamespace,
+) -> LogEvent {
     if let Some(hostname) = hostname {
-        event.insert(host_key, hostname.clone());
+        log_namespace.insert_source_metadata(
```

Comment:
Can we centralize this logic with the other `insert_source_metadata` calls?

Reply:
Theoretically, yes - I was just avoiding changing more of the code than necessary.

Reply:
#lazy 😅